### DISCUSSION TITLE
[Detector Support]: Jetson Orin Nano + Jetpack 6

### DISCUSSION THREAD

**Discussion by klaus7269** [blakeblackshear/frigate#15740] [Detector Support] [answered]
### Describe the problem you are having

Any plans on creating a jp6 image of frigate? 

Ps. Loving all your work, this system is fantastic. 

### Version

15

### Frigate config file

```yaml
Na
```

### docker-compose file or Docker CLI command

```yaml
Na
```

### Relevant Frigate log output

```shell
Na
```

### Install method

HassOS Addon

### Object Detector

TensorRT

### Screenshots of the Frigate UI's System metrics pages

Na

**Comment by NickM-27** [comment]
Jetson is a community supported board and the original contributor no longer is able to work on it, so we'll need a community member to contribute this feature.

**Comment by walthowd** [comment]
It looks like this will get easier to support JP6 on dev/0.16 based on bookworm and a more recent Python release:

https://github.com/blakeblackshear/frigate/pull/16390
https://github.com/blakeblackshear/frigate/pull/8914

cc @AndBobsYourUncle 

I am happy to take a look at some point -- possibly when the 15 release has stabilized and 16 starts to get a tiny bit stable. 

And as a total aside -- The devex experience on these NVIDIA Jetson boards is the absolute worst, what a pain for all the things.

**Comment by ramunasd** [comment]
As a jetson owner I already tried to refactor docker builds for more recent python versions. The biggest problem here is python minor version mismatch - most of frigate images are built on top of debian 12 which has python3.11 by default, while nvidia images based on ubuntu. The latter has either 3.10 for older and 3.12 for the more recent. Any attempt to install original nvidia packages on different python versions fails, so this is the biggest pain point IMO.

Anyways, I can help with nvidia docker builds and testing if support for jetson devices gets traction.

**Reply by hawkeye217** [reply, marked as answer]
A JP6 build was recently added to the dev branch and will be available for Frigate 0.16.

**Reply by ramunasd** [reply]
Thanks a lot, I will check.

**Reply by walthowd** [reply]
Is there a docker build out yet? I tried to peek but didn't see one.

**Reply by ramunasd** [reply]
Seems GHA cannot build due to segfaults and other weird errors, while local command `make local-trt-jp6` does the job.

**Reply by LanderN** [reply]
@NickM-27 fixed the docker build on GHA :rocket:

**Comment by LanderN** [comment]
Just FYI, I had to do this fix to get Frigate working on a freshly flashed Orin Nano with JetPack 6.2:
https://gist.github.com/jasongill/fef55cb769657bf36d0bb20ada742249

**Reply by jshumaker** [reply]
Is this something that has been baked into the image, or something one needs to do within the current image?  I'm attempting to run this on a new Orin Nano with JetPack 6.2.1 and with detector onnx it's not automatically detecting the hardware.

**Reply by NickM-27** [reply]
it would be something you do on the host

**Reply by jshumaker** [reply]
Hmm the file appears to now be included with 6.2.1 so this fix isn't needed there, but still not seeing the detector pick up the GPU.

**Reply by NickM-27** [reply]
you should create your own support discussion
